x-pack/plugins/actions/server/routes/execute.ts

```diff
@@ -49,16 +49,32 @@ export const executeActionRoute = (
       verifyApiAccess(licenseState);
       const { params } = req.body;
       const { id } = req.params;
-      const body: ActionTypeExecutorResult = await actionExecutor.execute({
-        params,
-        request: req,
-        actionId: id,
-      });
-      return body
-        ? res.ok({
-            body,
-          })
-        : res.noContent();
+      try {
+        const body: ActionTypeExecutorResult = await actionExecutor.execute({
+          params,
+          request: req,
+          actionId: id,
+        });
+        return body
+          ? res.ok({
+              body,
+            })
+          : res.noContent();
+      } catch (e) {
+        if (e instanceof ActionTypeDisabledError) {
+          switch (e.reason) {
```

Comment:
huh, since adding that new Error subtype, I was wondering if there'd be a case for having it handle the response.  Seems slightly weird, `ActionTypeDisabledError::sendResponse(res)` would do the appropriate `res.<whichError>(...)` kinda thing.  Kinda Boomish, I guess.

I see there's one more of these in the PR below, so ... maybe?

Reply:
I'll see what I can do, not a bad idea!

Reply:
Done in https://github.com/elastic/kibana/pull/59153/commits/9af1336246b2d62be7e171acf64aa9fc8fc644d7